components/sections/Project.jsx

```diff
@@ -2,19 +2,33 @@ import Image from "next/image";
 import Link from "next/link";
 import { useState } from "react";
 import { projectData } from "../../utils/projectData";
-
+import { useRef, useEffect } from "react";
 export default function Project() {
   const [shown, setShown] = useState(0);
+  const [scrolled, setScrolled] = useState(false);
+
+  const scrollSticked = () => {
+    console.log(window.scrollY);
+    window.scrollY > 1380 ? setScrolled(true) : setScrolled(false);
+  };
+  useEffect(() => {
+    window.addEventListener("scroll", scrollSticked);
+  }, [scrolled]);
+
```

Comment:
**Just a suggestion**: I hear that when working with event listeners in useEffect it is best to `clean up` (remove)  those event listeners after the component mounts
something like this 
```js
useEffect(() => {
  const handleClick = ( ) => {
    console.log('Button clicked');
  };
  element.addEventListener('click', handleClick);
  return () => {
    element.removeEventListener('click', handleClick);
  };
}, []);
```
I don't really understand the concept so I don't know if we need it here. are you familiar with the concept? if yes do you think it is something we can implement in our case?

PS: if what I said was not clear check this link https://www.pluralsight.com/guides/how-to-cleanup-event-listeners-react
I'm talking about the same thing